commons-test.mk

```diff
@@ -12,6 +12,7 @@ test-%:
 		--packages="./..." \
 		--junitfile TEST-$*.xml \
 		-- \
+                -coverprofile=coverage.out ./... \
```

Comment:
Will be the `./...` irrelevant given we have https://github.com/testcontainers/testcontainers-go/pull/1518/files#diff-f18f81daf1f8305a66be78cac5d122d165f39391f0348477f133858018585cbcR12 in place?

Taking a look at how gotestsum explains it: https://github.com/gotestyourself/gotestsum#custom-go-test-command, it's not clear to me if they are duplicated without side effects

Reply:
I can remove it in a first time. We’ll see if it works ;) !